src/ports/postgres/modules/deep_learning/madlib_keras_model_selection.py_in

```diff
@@ -254,13 +261,17 @@ class MstSearch():
                  object_table=None,
                  **kwargs):
 
+        self.schema_madlib = schema_madlib
         self.model_arch_table = model_arch_table
         self.model_selection_table = model_selection_table
         self.model_selection_summary_table = add_postfix(
             model_selection_table, "_summary")
         self.model_id_list = sorted(list(set(model_id_list)))
 
+        if object_table is not None and schema_madlib not in object_table:
+            object_table = "{0}.{1}".format(schema_madlib, object_table)
```

Comment:
This also looks insecure.
Let's add this test as well:

  `object_table="my_insecure_schema.madlib_vulnerability"`